DSharpPlus.Commands/Trees/CommandParameterBuilder.cs

```diff
@@ -116,8 +116,11 @@ public static CommandParameterBuilder From(ParameterInfo parameterInfo)
             commandParameterBuilder.WithDefaultValue(parameterInfo.DefaultValue);
         }
 
-        // Might be set by the `ArgumentAttribute`
-        if (string.IsNullOrWhiteSpace(commandParameterBuilder.Name) && !string.IsNullOrWhiteSpace(parameterInfo.Name))
+        if (parameterInfo.GetCustomAttribute<ParameterAttribute>() is { } attribute)
```

Comment:
Nit: `is ParameterAttribute parameterAttribute`

Reply:
equivalent, but if you want

Reply:
For consistency. And honestly this is the pattern matching equivalent of `var` imo.